contracts/src/storage/AssetsStorage.sol

```diff
@@ -18,6 +19,14 @@ library AssetsStorage {
         uint256 registerTokenFee;
         // Foreign token registry by token ID
         mapping(bytes32 foreignID => address) tokenAddressOf;
+        uint8 foreignTokenDecimals;
+        // The maximum fee that can be sent to a destination parachain to pay for execution (DOT).
+        // Has two functions:
+        // * Reduces the ability of users to perform arbitrage using a favourable exchange rate
+        // * Prevents users from mistakenly providing too much fees, which would drain AssetHub's
+        //   sovereign account here on Ethereum.
+        uint128 maxDestinationFee;
+        address weth;
```

Comment:
Move WETH to the first item. I will be adding this as part of the AutoWrapping/Unwrapping feature. So this change will help migration got smoothly.
```suggestion
        address weth;
        uint8 foreignTokenDecimals;
        // The maximum fee that can be sent to a destination parachain to pay for execution (DOT).
        // Has two functions:
        // * Reduces the ability of users to perform arbitrage using a favourable exchange rate
        // * Prevents users from mistakenly providing too much fees, which would drain AssetHub's
        //   sovereign account here on Ethereum.
        uint128 maxDestinationFee;
```

https://github.com/Snowfork/snowbridge/pull/1343/files#diff-3c18b55dea02438f70b3d8fe9ddfb126767f57a5eb4b6840d884c8883f73f521R21-R22